realm/realm-library/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -925,6 +926,42 @@ public String toString() {
         return sb.toString();
     }
 
+    /**
+     * Returns {@link RealmResults} containing all {@code srcClassName} class objects that have a relationship
+     * to this object from {@code srcFieldName} field.
+     *
+     * @param srcClassName name of the class returned objects belong to.
+     * @param srcFieldName name of the field to use. Field type must be either {@code io.realm.RealmFieldType.OBJECT} or
+     * {@code io.realm.RealmFieldType.LIST}.
+     * @return the result.
+     */
+    public RealmResults<DynamicRealmObject> linkingObjects(String srcClassName, String srcFieldName) {
+        final DynamicRealm realm = (DynamicRealm) proxyState.getRealm$realm();
+        realm.checkIfValid();
+        proxyState.getRow$realm().checkIfAttached();
+
+        final RealmSchema schema = realm.getSchema();
+        final RealmObjectSchema realmObjectSchema = schema.get(srcClassName);
+        if (realmObjectSchema == null) {
+            throw new IllegalArgumentException("Class not found: " + srcClassName);
+        }
+
+        if (srcFieldName != null && srcFieldName.contains(".")) {
+            throw new IllegalArgumentException(MSG_LINK_QUERY_NOT_SUPPORTED);
```

Comment:
There, really, is no need for defining constants for strings, like this.  They will be interned on compile...